nukkit.jar.conf

```
@@ -1,6 +1,6 @@
 [config]
 name = NukkitX
-source = https://ci.nukkitx.com/job/NukkitX/job/Nukkit/job/master/lastSuccessfulBuild/artifact/target/nukkit-1.0-SNAPSHOT.jar
+source = https://ci.opencollab.dev/job/NukkitX/job/Nukkit/job/master/lastSuccessfulBuild/artifact/target/nukkit-1.0-SNAPSHOT.jar
 configSource = https://raw.githubusercontent.com/NukkitX/multicraft/master/nukkit.jar.conf
```

RE above comment change this to `CloudburstMC`
```suggestion
configSource = https://raw.githubusercontent.com/CloudburstMC/multicraft/master/nukkit.jar.conf
```